docs/framework/wcf/feature-details/how-to-use-a-custom-user-name-and-password-validator.md

```diff
@@ -39,7 +39,7 @@ For a sample application, see [User Name Password Validator](../samples/user-nam
     When using transport-level security over HTTP(S), add either the [\<wsHttpBinding>](../../configure-apps/file-schema/wcf/wshttpbinding.md) or [\<basicHttpBinding>](../../configure-apps/file-schema/wcf/basichttpbinding.md), a [\<netTcpBinding>](../../configure-apps/file-schema/wcf/nettcpbinding.md) or a [\<customBinding>](../../configure-apps/file-schema/wcf/custombinding.md) that uses HTTP(S) and the `Basic` authentication scheme.
 
     > [!NOTE]
-    > When [!INCLUDE[netfx35_long](../../../../includes/netfx35-long-md.md)] or later is used, you can use a custom username and password validator with message and transport security. With WinFX, a custom username and password validator can only be used with message security.
+    > When .NET Framework version 3.5 or later is used, you can use a custom username and password validator with message and transport security. With WinFX, a custom username and password validator can only be used with message security.
```

Comment:
```suggestion
    > When .NET Framework 3.5 or later version is used, you can use a custom username and password validator with message and transport security. With WinFX, a custom username and password validator can only be used with message security.
```

Any suggestion for the "version X or later" pattern?

Reply:
We usually say .NET Framework 3.5 or later versions